switch.xsh

```diff
@@ -0,0 +1,10 @@
+if 'please.go' in @(ls):
```

Comment:
Should we add some sort of usr bin env xonsh here and make the script executable?